### DISCUSSION TITLE
Flameshot takes screen screenshot, not area

### DISCUSSION THREAD

**Discussion by somyk33** [flameshot-org/flameshot#2870] [Q&A] [answered]
When I'm trying to take a screenshot, it takes all screen and then I have to go in options, select option "select area to grab" and click "take screenshot".
[YT vid](https://youtu.be/EecIK9Ze-uA)

**Comment by mmahmoudian** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/390889/184443725-bdd66a2a-0b06-46b6-b052-8ff5031ceff8.png)
